packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -261,16 +261,16 @@ The entity upon which an actor performs an action. Targets may be anything: an a
-| geographicalContext | The physical location where the client made its request from                                                                                                                                          | [GeographicalContext object](#geographicalcontext-object) | TRUE     |
+| id                  | For OAuth requests, this is the ID of the OAuth [client](https://tools.ietf.org/html/rfc6749#section-1.1) making the request. For SSWS token requests, this is the ID of the agent making the request. | String                                                    | TRUE     |
+| userAgent           | The [user agent](https://en.wikipedia.org/wiki/User_agent) that is used by an actor to perform an action                                                                                                      | [UserAgent object](#useragent-object)                     | TRUE     |
+| geographicalContext | The physical location where the client is making its request from                                                                                                                                          | [GeographicalContext object](#geographicalcontext-object) | TRUE     |
 | zone                | The `name` of the [Zone](/docs/reference/api/zones/#ZoneModel) that the client's location is mapped to                                                                                                | String                                                    | TRUE     |
-| ipAddress           | Ip address that the client made its request from                                                                                                                                                      | String                                                    | TRUE     |
-| device              | Type of device that the client operated from (e.g. Computer)                                                                                                                                          | String                                                    | TRUE     |
+| ipAddress           | IP address that the client is making its request from                                                                                                                                                      | String                                                    | TRUE     |
+| device              | Type of device that the client operates from (for example, Computer)                                                                                                                                          | String                                                    | TRUE     |
 
 ### UserAgent object
 
```

Comment:
Curious usage here starting the content off with a quote from wikipedia. Might be a team discussion.

Reply:
In the previous client object section, the object was set off with code style `client`. Shouldn't we use the same formatting for the UserAgent object here?